src/App/controllers/blog.js

```diff
@@ -64,13 +82,42 @@ class BlogControllers {
 
   static async updateBlog(req, res, next) {
     try {
+      let result;
+      if (req.files) {
```

Comment:
Can you re-use the middleware you already have instead of duplicating your codes?

Reply:
Done✔